### ISSUE TITLE
networking in windows 98 is limited only to navigating internet

### ISSUE THREAD

**Issue by gfernval** [joncampbell123/dosbox-x#5074]
### Is your feature request related to a problem? Please describe.

There is no way to establish any form of communication between host operating system running dosbox-x (Windows 7 for example) and operating system running inside dosbox-x (Windows 98), when running Windows 98 inside dosbox-x (and configuring network card accordingly) you can only navigate Internet and cannot pass data back and forth between Windows 7 running dosbox-x and Windows 98 running inside dosbox-x in any way

### What you want

A mechanism to pass data between host operating system (Windows 7 for example) and operating system running inside dosbox-x, like the virtual box guest additions and installs a share inside the operating system running inside virtualbox

### Have you checked that no similar feature request(s) exist?

- [X] I have searched and didn't find any similar feature request.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by gfernval** [comment]
It is possible to establish a communication between Windows 7 host and Windows 98 dosbox-x guest? All the solutions depicted (samba, ftp servers, Microsoft client, netbeui, shared folders,,..) only work if pinging inside Windows 98 (running under dosbox-x) to the host running dosbox-x (windows 7 in my case) gives a reply (does not give a reply), pinging inside Windows 98 to the gateway (192.168.1.1) gives a reply, but pinging to the Windows 7 host (192.168.1.150) does not give a reply. Windows 98 has gotten the ip 192.168.1.151 through DHCP) I have use the Microsoft  compiled latest version of dosbox-x

**Comment by gfernval** [comment]
I have used the pcap method, works in order to access internet inside Windows 98 under dosbox-x

**Comment by gfernval** [comment]
Taking another different route, editing dosbox-x.conf and adding at the end of the file the lines:

IMGMOUNT C Win98SE.img
MOUNT E c:\utils
BOOT C:

If I copy files from inside Windows 98 to the E: drive letter created by MOUNT E c:\utils, when I exit Windows 98 such files copied does not appear in c:\utils, what I´m doing wrong?

**Comment by m-p-3** [comment]
@gfernval 
> If I copy files from inside Windows 98 to the E: drive letter created by MOUNT E c:\utils, when I exit Windows 98 such files copied does not appear in c:\utils, what I´m doing wrong?

https://dosbox-x.com/wiki/Guide%3AInstalling-Windows-98#_host_folder_mounts

> This conversion is dynamic, effectively creating an emulated FAT harddisk out of a folder mount. This converted FAT harddisk is not automatically stored permanently.

TL;DR: the change is made in memory, and you have to store/comment the changes made to an image file. The guest OS is unable to directly write to the mounted folder from what I understand.

However I stumbled upon this

https://dosbox-x.com/wiki/#Home#_frequently_asked_questions_faq

> **Can DOSBox-X synchronize the content of mounted local drives with the host file system and/or the internal clock with the host system clock automatically?**
> Note that you can also use `RESCAN` command to manually synchronize the content of mounted local drive(s) with the host file system (type `RESCAN /?` for more information about this command)

However I'm not sure how I can run that command without rebooting the guest OS and losing the changes made.

**Comment by rderooy** [comment]
No, the rescan will not help you. The command will not be available, and would not work anyway.
